package.json

```diff
@@ -42,6 +46,7 @@
     "eth-json-rpc-middleware": "^8.0.1",
     "eth-sig-util": "^3.0.1",
     "ethers": "=5.6.1",
+    "faunadb": "^4.5.2",
```

Comment:
Moved `faunadb` to `dependencies` to prevent lint warnings in `.js` files under the `fauna` directory.